tests/Integrations/CLI/CakePHP/V2_8/CommonScenariosTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\CLI\CakePHP\V2_8;
+
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Integrations\CLI\CLITestCase;
+
+final class CommonScenariosTest extends CLITestCase
+{
+    protected function getScriptLocation()
+    {
+        return __DIR__ . '/../../../../Frameworks/CakePHP/Version_2_8/app/Console/cake.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'DD_SERVICE_NAME' => 'cake_console_test_app',
+        ]);
+    }
+
+    public function testCommandWithNoArguments()
+    {
+        $traces = $this->getTracesFromCommand();
+
+        $this->assertSpans($traces, [
+            SpanAssertion::build(
+                'cakephp.console',
+                'cake_console_test_app',
+                'cli',
+                'cake_console'
+            )->withExactTags([
+                'integration.name' => 'cakephp',
+            ])
+        ]);
+    }
+
+    public function testCommandWithArgument()
+    {
+        $traces = $this->getTracesFromCommand('command_list');
+
+        $this->assertSpans($traces, [
+            SpanAssertion::build(
+                'cakephp.console',
```

Comment:
Do we want to trace cake console by default?

Reply:
All CLI integrations are disabled by default but this will be enabled by setting `DD_TRACE_CLI_ENABLED=true`. :)